WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift

```diff
@@ -0,0 +1,323 @@
+    private let reblogButton = UIButton()
+    private let commentButton = UIButton()
+    private let likeButton = UIButton()
+    private let fillerView = UIView()
+    private let menuButton = UIButton()
+
+    private let separatorView = UIView()
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        commonInit()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        commonInit()
+    }
+
+}
+
+// MARK: - Private methods
+
+private extension ReaderPostCardCell {
+
+    func commonInit() {
+        setupViews()
+        addViewConstraints()
+    }
+
+    // MARK: - View setup
+
+    func setupViews() {
+        setupContentView()
+        setupContentStackView()
+
+        setupIconImage(avatarImageView,
+                       containerView: avatarContainerView,
+                       image: Constants.avatarPlaceholder)
+        setupIconImage(siteIconImageView,
+                       containerView: siteIconContainerView,
+                       image: Constants.siteIconPlaceholder)
+        setupSiteTitle()
+        setupPostDate()
+        setupSiteStackView()
+
+        setupPostTitle()
+        setupPostSummary()
+        setupFeaturedImage()
+        setupPostCounts()
+
+        setupControlButtons()
+        setupControlsStackView()
+
+        setupSeparatorView()
+    }
+
+    func setupContentView() {
+        contentView.backgroundColor = .listForeground
+    }
+
+    func setupContentStackView() {
+        contentStackView.translatesAutoresizingMaskIntoConstraints = false
+        contentStackView.axis = .vertical
+        contentStackView.alignment = .leading
+        contentStackView.spacing = Constants.ContentStackView.spacing
+        contentView.addSubview(contentStackView)
+    }
+
+    func setupIconImage(_ imageView: UIImageView, containerView: UIView, image: UIImage?) {
+        containerView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.layer.cornerRadius = Constants.iconImageSize / 2.0
+        imageView.layer.masksToBounds = true
+        imageView.image = image
+        containerView.addSubview(imageView)
+        siteStackView.addArrangedSubview(containerView)
+    }
+
+    func setupSiteTitle() {
+        siteTitleLabel.translatesAutoresizingMaskIntoConstraints = false
+        siteTitleLabel.font = .preferredFont(forTextStyle: .subheadline).semibold()
+        siteTitleLabel.numberOfLines = 1
+        siteTitleLabel.setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
+        siteTitleLabel.setText("Site Title") // TODO: Remove
+        siteStackView.addArrangedSubview(siteTitleLabel)
+    }
+
+    func setupPostDate() {
+        postDateLabel.translatesAutoresizingMaskIntoConstraints = false
+        postDateLabel.font = .preferredFont(forTextStyle: .subheadline)
```

Comment:
I think the post date should be in `.footnote`:

<img width="189" alt="Screenshot 2023-10-06 at 23 10 48" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/1299411/d7df6f27-f4e5-47cf-9643-9d7f952de01b">

Also, although it won't be addressed here, keep in mind about the RTL behavior since the site title and the post date are separated by the `•` character.

Reply:
I updated the font to be `.footnote` in https://github.com/wordpress-mobile/WordPress-iOS/pull/21685/commits/14915575ac089f78ef79fcb76cdb4d6e0ad6fe90. With the RTL behavior, that is something I need to fix, but I'll do it in the follow up PRs.